products.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+require_once 'src/Factories/PdoFactory.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Services/ProductListDisplayService.php';
+require_once 'src/Models/CategoryModel.php';
+require_once 'src/Entities/Product.php';
+
+$categoryTitle = 'Oops! Invalid product ID';
+$productList = [];
```

Comment:
set this to false by default as thats what the fetch will return if theres no data in the db, keeps it consistent